packages/@okta/vuepress-site/docs/guides/set-up-event-hook/implement-your-service/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Implement Your Service
+---
+
+You need to implement a web service with an Internet-accessible endpoint to receive event hook calls from Okta. It's your responsibility to develop the code and to arrange its hosting on a system external to Okta. Okta defines the REST API contract for the REST requests it sends to your service.
```

Comment:
I think this first sentence is missing something along the lines of "This guide is intended for developers who need to..."